manager/controlapi/ca_rotation.go

```diff
@@ -0,0 +1,216 @@
+package controlapi
+
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+
+	"github.com/cloudflare/cfssl/helpers"
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/ca"
+	"github.com/docker/swarmkit/log"
+)
+
+var minRootExpiration = helpers.OneYear
```

Comment:
I also don't feel strongly - happy to change it to 3 years too.  I just picked some value.  :)